README.md

```diff
@@ -81,31 +103,69 @@ For, Lisk Sepolia Testnet:
 
 ```sh
 ./build/bin/geth \
-  --http \
-  --http.port=8545 \
-  --http.addr=localhost \
-  --authrpc.addr=localhost \
-  --authrpc.jwtsecret=./jwt.txt \
-  --verbosity=3 \
-  --rollup.sequencerhttp=https://rpc.sepolia-api.lisk.com/ \
-  --op-network=lisk-sepolia \
-  --datadir=$DATADIR_PATH \
-  --override.canyon=0
+    --datadir=$DATADIR_PATH \
+    --verbosity=3 \
+    --http \
+    --http.corsdomain="*" \
+    --http.vhosts="*" \
+    --http.addr=0.0.0.0 \
```

Comment:
Let's default to using `127.0.0.1` instead of `0.0.0.0` everywhere.
The end-user can decide if they want to bind the app on all the available network interfaces.